app/src/main/java/org/oppia/android/app/help/HelpActivity.kt

```diff
@@ -59,12 +68,17 @@ class HelpActivity :
     val selectedLicenseIndex = savedInstanceState?.getInt(LICENSE_INDEX_SAVED_KEY) ?: 0
     selectedHelpOptionsTitle = savedInstanceState?.getStringFromBundle(HELP_OPTIONS_TITLE_SAVED_KEY)
       ?: resourceHandler.getStringInLocale(R.string.faq_activity_title)
+    val policyPage = savedInstanceState?.getProto(
+      POLICIES_ARGUMENT_PROTO,
+      PoliciesActivityArguments.getDefaultInstance()
+    )
     helpActivityPresenter.handleOnCreate(
       selectedHelpOptionsTitle,
       isFromNavigationDrawer,
       selectedFragment,
       selectedDependencyIndex,
-      selectedLicenseIndex
+      selectedLicenseIndex,
+      policyPage
```

Comment:
This is named as `policiesActivityArguments` in `HelpActivityPresenter` and therefore I suggest naming it same here too. Update key name and its value too accordingly.

Reply:
Done.